packages/ng/modal/src/lib/modal-panel.component.ts

```diff
@@ -38,8 +42,12 @@ export abstract class ALuModalPanelComponent<T extends ILuModalContent> implemen
 	public readonly modalId = modalId++;
 
 	private _subs = new Subscription();
+	public modalClasses: LuModalClasses;
 
-	constructor(protected _ref: ALuModalRef<T>, protected _cdr: ChangeDetectorRef) {}
+	constructor(protected _ref: ALuModalRef<T>, protected _cdr: ChangeDetectorRef, _elementRef: ElementRef<HTMLElement>, _renderer: Renderer2) {
```

Comment:
```suggestion
	constructor(protected _ref: ALuModalRef<T>, _elementRef: ElementRef<HTMLElement>, _renderer: Renderer2) {
```

`ChangeDetectorRef` isn't needed anymore (yay!)

Reply:
Oh thx!